packages/yourloops/test/components/dashboard-widgets/medical-files/weekly-reports-dialog.test.tsx

```diff
@@ -0,0 +1,85 @@
+/**
```

Comment:
Some tests are missing, ping me to discuss it :slightly_smiling_face:

Reply:
ping